src/scripts/crowdin-import.js

```diff
@@ -0,0 +1,369 @@
+const { copyFileSync, existsSync, mkdirSync, readdirSync } = require("fs")
+const { resolve, join } = require("path")
+
+/**
+ * 0. Console summary flags
+ */
+const VERBOSE = false
+const FULL_BUCKET_NAME_SUMMARY = false
```

Comment:
Would be nice if we could pass these params as args to the script. You could install something like https://www.npmjs.com/package/minimist to help you with that